db/schema.rb

```diff
@@ -418,6 +420,18 @@
     t.index ["parent_id"], name: "index_migration_failures_on_parent_id"
   end
 
+  create_table "milestones", force: :cascade do |t|
+    t.bigint "schedule_id"
+    t.enum "declaration_type", null: false, enum_type: "declaration_types"
+    t.date "start_date", null: false
+    t.date "payment_date", null: false
```

Comment:
I don't think we use `payment_date` in ECF1, I've double checked, so I think we can drop :)